sdk/eventhubs/azure-messaging-eventhubs/vcpkg/vcpkg.json

```diff
@@ -14,7 +14,7 @@
     {
       "name": "azure-core-amqp-cpp",
       "default-features": false,
-      "version>=": "1.0.0-beta.2"
+      "version>=": "1.0.0-beta.5"
```

Comment:
We haven't shipped this version to vcpkg yet, shouldn't this be 1.0.0-beta.4?
https://github.com/Azure/azure-sdk-for-cpp/blob/main/sdk/core/azure-core-amqp/CHANGELOG.md